docs/most-common-issues.md

```diff
@@ -0,0 +1,14 @@
+Before you do anything else try to remove the project again from your computer and clone it again
+
+```bash
+rm -rf [yourpath]/debugger.html
```

Comment:
This is not working on windows and we should avoid just pointing out platform specific solutions.

Reply:
should we suggest `yarn nom` somewhere.
also, perhaps checkout master... and git pull --rebase